pallets/pips/src/lib.rs

```diff
@@ -1426,32 +1422,6 @@ impl<T: Trait> Module<T> {
             .collect::<Vec<_>>()
     }
 
-    /// Retrieve historical voting of `who` account.
-    pub fn voting_history_by_address(
-        who: T::AccountId,
-    ) -> HistoricalVotingByAddress<Vote<BalanceOf<T>>> {
-        <Proposals<T>>::iter()
-            .filter_map(|(_, pip)| {
-                Some(VoteByPip {
-                    pip: pip.id,
-                    vote: Self::proposal_vote(pip.id, &who)?,
-                })
-            })
-            .collect::<Vec<_>>()
-    }
-
-    /// Retrieve historical voting of `who` identity.
-    /// It fetches all its keys recursively and it returns the voting history for each of them.
-    pub fn voting_history_by_id(
-        who: IdentityId,
-    ) -> HistoricalVotingById<T::AccountId, Vote<BalanceOf<T>>> {
-        let flatten_keys = <Identity<T>>::flatten_keys(who, 1);
```

Comment:
Looks like `pub fn flatten_keys(id: IdentityId, max_depth: u8) -> Vec<T::AccountId> {` can be nuked too now.